### PR TITLE
GH-100 Add kick command example.

### PR DESCRIPTION
**Pull request by Skyy0x**
An example of a command to learn about use Joiner.

### CODE REVIEW

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.argument.Name;
+import org.bukkit.entity.Player;
+import org.bukkit.Bukkit;
+import dev.rollczi.litecommands.argument.joiner.Joiner;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.section.Section;
+
+@Section(route = "fuckoff", aliases = "wypierdalaj")
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:13]: kick

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@Permission("dev.rollczi.fuckoff")
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:14]: dev.rollczi.kick

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+public class FuckOffCommand {
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:15]: KickCommand

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    @Execute(min = 2)
+    public void banPlayer(final Player sender, @Arg @Name("target") final Player target, @Joiner final String reason){
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:18]: banPlayer -> kickPlayer

Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:18]:
```suggestion
    void kickPlayer(@Arg @Name("target") Player target, @Joiner String reason) {

```

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        target.kickPlayer(ChatUtil.color(reason));
+        Bukkit.broadcastMessage(ChatUtil.color("&cPlayer &7" + target.getName() + " &cgot fucked up!"));
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:20]: Use Server#broadcastMessage

**Review by Rollczi — CHANGES_REQUESTED** [review]

**Review by Skyy0x — COMMENTED** [review]
changes

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import dev.rollczi.litecommands.command.section.Section;
+
+@Section(route = "kick")
+@Permission("dev.rollczi.kick")
+public class KickCommand {
+
+    @Execute(min = 2)
+    public void kickPlayer(Player sender, @Arg @Name("target") Player target, @Joiner String reason){
+        target.kickPlayer(ChatUtil.color(reason));
+        Bukkit.broadcastMessage(ChatUtil.color("&cPlayer &7" + target.getName() + " &chas been kicked!"));
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:20]: Użyj Server#broadcastMessage

File: examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.section.Section;
+
+@Section(route = "kick")
+@Permission("dev.rollczi.kick")
+public class KickCommand {
+
+    @Execute(min = 2)
+    public void kickPlayer(Player sender, @Arg @Name("target") Player target, @Joiner String reason){
```
Comment (Rollczi) [inline examples/bukkit/src/main/java/dev/rollczi/example/bukkit/FuckOffCommand.java:18]: `( {`

**Review by Rollczi — CHANGES_REQUESTED** [review]

**Review by Rollczi — APPROVED** [review]

**Review by vLuckyyy — APPROVED** [review]

**Review by Rollczi — APPROVED** [review]

**Pull request merged**